### PR TITLE
Group interaction stories and document using multiple modifier keys

### PR DESCRIPTION
**Pull request by axelboc**
- Group all the stories under `Building Blocks/Interactions`
- Add stories to demonstrate setting multiple modifier keys
- Move `AxialSelectToZoom` stories into separate file
- Refactor stories (default `args`, interaction props types, checkboxes for `modifierKey` controls, etc.)

### CODE REVIEW

File: apps/storybook/src/AxialSelectToZoom.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <AxialSelectToZoom {...args} />
+      <ResetZoomButton />
+
+      <DataCurve
+        abscissas={getAxisValues(undefined, mockValues.oneD.length)}
+        ordinates={mockValues.oneD}
+        errors={mockValues.oneD_errors}
+        color="blue"
+        showErrors
+      />
```
Comment (axelboc) [inline apps/storybook/src/AxialSelectToZoom.stories.tsx:40]: Adding a curve so the zoom behaviour is clearer, like the image in `SelectToZoom`.

File: apps/storybook/src/AxialSelectToZoom.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    </VisCanvas>
+  );
+};
+
+export const DefaultX = Template.bind({});
+DefaultX.args = { axis: 'x' };
+
+export const DefaultY = Template.bind({});
+DefaultY.args = { axis: 'y' };
```
Comment (axelboc) [inline apps/storybook/src/AxialSelectToZoom.stories.tsx:49]: No need for custom args -- I'm using the props of `AxialSelectToZoom` directly.

File: apps/storybook/src/DefaultInteractions.stories.tsx [hunk 3]
```diff
@@ -19,19 +19,19 @@ const Template: Story<DefaultInteractionsProps> = (args) => {
...
-  argTypes: {
-    keepRatio: { defaultValue: false },
-    pan: { defaultValue: {} },
-    zoom: { defaultValue: {} },
-    xAxisZoom: { defaultValue: { modifierKey: 'Alt' } },
-    yAxisZoom: { defaultValue: { modifierKey: 'Shift' } },
-    selectToZoom: { defaultValue: { modifierKey: 'Control' } },
-    xSelectToZoom: { defaultValue: { modifierKey: ['Control', 'Alt'] } },
-    ySelectToZoom: { defaultValue: { modifierKey: ['Control', 'Shift'] } },
-    withAxialZoom: { defaultValue: true },
```
Comment (axelboc) [inline apps/storybook/src/DefaultInteractions.stories.tsx:35]: This syntax is deprecated.

File: apps/storybook/src/XAxisZoom.stories.tsx [hunk 4]
```diff
@@ -23,36 +18,35 @@ const Template: Story<TemplateProps> = (args) => {
...
-    controls: {
-      include: ['disabled', 'modifierKey'],
-    },
+  title: 'Building Blocks/Interactions/XAxisZoom',
+  component: XAxisZoom,
+  parameters: { layout: 'fullscreen' },
+  decorators: [FillHeight],
+  args: {
+    modifierKey: [],
+    disabled: false,
```
Comment (axelboc) [inline apps/storybook/src/XAxisZoom.stories.tsx:44]: Setting default `args` even for optional props like `disabled` helps with ordering the controls in the Storybook UI.

File: apps/storybook/src/YAxisZoom.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import { Pan, ResetZoomButton, VisCanvas, YAxisZoom } from '@h5web/lib';
```
Comment (axelboc) [inline apps/storybook/src/YAxisZoom.stories.tsx:1]: `YZoom.stories.tsx` renamed to `YAxisZoom` (same as `XZoom.stories.tsx` above but here, git didn't notice).

**Review by loichuder — APPROVED** [review]

**Pull request merged**